### PR TITLE
Feat, add /var/lib/lxcfs remount when lxc service restart

### PR DESCRIPTION
**Pull request by dongjiang1989**
### What type of PR is this?
/kind feature

- [x] Unittest
- [x] Coverage 80%+
- [x] Build Success

### What this PR does / why we need it:
Add lxcfs remount script

### CODE REVIEW

File: hack/script/container_remount_lxcfs.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#! /bin/bash
+
+PATH=$PATH:/bin
+LXCFS="/var/lib/lxc/lxcfs"
+LXCFS_ROOT_PATH="/var/lib/lxc"
```
Comment (imlense) [inline hack/script/container_remount_lxcfs.sh]: PATH error
```bash
LXCFS="/var/lib/lxcfs"
LXCFS_ROOT_PATH="/var/lib"
```

File: hack/script/lxcfs.service [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ConditionVirtualization=!container
+Before=lxc.service
+Documentation=man:lxcfs(1)
+
+[Service]
+ExecStart=/usr/bin/lxcfs -l /var/lib/lxc/lxcfs/
+KillMode=process
+Restart= always
+Delegate=yes
+ExecStopPost=-/bin/fusermount -u /var/lib/lxc/lxcfs
```
Comment (imlense) [inline hack/script/lxcfs.service]:
```shell
ExecStopPost=-/bin/fusermount -u /var/lib/lxcfs
```

File: hack/script/lxcfs.service [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=FUSE filesystem for LXC
+ConditionVirtualization=!container
+Before=lxc.service
+Documentation=man:lxcfs(1)
+
+[Service]
+ExecStart=/usr/bin/lxcfs -l /var/lib/lxc/lxcfs/
```
Comment (imlense) [inline hack/script/lxcfs.service]: ExecStart=/usr/bin/lxcfs -l /var/lib/lxcfs/

File: hack/script/container_remount_lxcfs.sh (hunk 1 above)
Comment (dongjiang1989) [reply hack/script/container_remount_lxcfs.sh]: done

File: hack/script/lxcfs.service (hunk 2 above)
Comment (dongjiang1989) [reply hack/script/lxcfs.service]: Done

File: hack/script/lxcfs.service (hunk 3 above)
Comment (dongjiang1989) [reply hack/script/lxcfs.service]: Done

**Comment by imlense** [comment]
LGTM!

**Review by imlense — APPROVED** [review]

**Pull request merged**